beacon-chain/p2p/gossip_topic_mappings.go

```diff
@@ -21,12 +23,33 @@ var GossipTopicMappings = map[string]proto.Message{
 	SyncCommitteeSubnetTopicFormat:            &prysmv2.SyncCommitteeMessage{},
 }
 
+// GossipTopicMappings is a function to return the assigned data type
+// versioned by epoch.
+func GossipTopicMappings(topic string, epoch types.Epoch) proto.Message {
+	if topic == BlockSubnetTopicFormat && epoch >= params.BeaconConfig().AltairForkEpoch {
+		return &prysmv2.SignedBeaconBlock{}
+	}
+	return gossipTopicMappings[topic]
```

Comment:
Possibility of panic on not found? If impossible, no worries about it

Reply:
we do check for nil objects returned for any use of this method, so it is handled in a sense. Although this can be made better.